### PR TITLE
Create roadmap view

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- create roadmap view:

**Attachments:**
- capabilities view on desktop:
image

- capabilities view on mobile:
image

- roadmap view on desktop:
image

- roadmap view on mobile:
image

- modal view on desktop:
image

- modal view on mobile: 
image

**Related issue(s)**
Epic: https://github.com/kyma-project/website/issues/258
Resolved https://github.com/kyma-project/website/pull/280
Resolved https://github.com/kyma-project/website/issues/279

### CODE REVIEW

File: src/components/layout/Layout.tsx [hunk 1]
```diff
@@ -54,13 +56,15 @@ const Layout: React.FunctionComponent<LayoutProps> = ({
...
-                <CookiesBanner />
-                {/* {backToTopButton && <BackToTop />} */}
-                <Header horizontalBg={horizontalHeaderBg} />
-                <Content>{children}</Content>
-                <Footer />
+                <ModalProvider>
+                  <SlidesBanner {...slidesProps} />
+                  <Popup />
+                  <CookiesBanner />
+                  {/* {backToTopButton && <BackToTop />} */}
```
Comment (akucharska) [inline src/components/layout/Layout.tsx]: Delete if unnecessary.

File: src/components/layout/Navigation.tsx [hunk 2]
```diff
@@ -27,6 +27,10 @@ const navigation = [
     path: "/blog/",
     title: "Blog",
   },
+  // {
```
Comment (akucharska) [inline src/components/layout/Navigation.tsx]: Delete if unnecessary

File: src/components/roadmap/Dropdown/styled.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 1;
+  width: 320px;
+  background-color: #fff;
+  border-radius: 8px;
+  box-shadow: 0 2px 26px 0 rgba(11, 116, 222, 0.2);
+  padding: 26px;
+  max-height: 700px;
+  /* overflow-y: scroll;
```
Comment (akucharska) [inline src/components/roadmap/Dropdown/styled.ts]: Remove comments

File: src/components/roadmap/Filters/Filters.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+import { FiltersList, Filter, FilterButton } from "./styled";
+
+const Filters: React.FunctionComponent = () => {
+  const { capabilities } = useContext(RoadmapService);
+  const { filters, setCapability, clearFilters } = useContext(TicketsService);
+
+  return (
+    <>
+      {filters.capabilities.length ? (
```
Comment (akucharska) [inline src/components/roadmap/Filters/Filters.tsx]: if(!filters.capabilities.length) {return null;}

return <FiltersList>...</FiltersList>

File: package.json [hunk 5]
```diff
@@ -32,7 +32,8 @@
     "publish:fork": "mv static/CNAME ./CNAME_bak && npm run build -- --prefix-paths && gh-pages -d public -m 'Deploy new version of site' -o origin && mv ./CNAME_bak static/CNAME",
     "lint-check": "tslint -c tslint.json 'src/**/*.{ts,tsx}' --format verbose && prettier --check '**/*.{ts,tsx,js,jsx,json,html,css,yaml}'",
     "lint-fix": "tslint -c tslint.json 'src/**/*.{ts,tsx}' --format verbose --fix && prettier --write '**/*.{ts,tsx,js,jsx,json,html,css,yaml}'",
-    "conflict-check": "tslint-config-prettier-check ./tslint.json"
+    "conflict-check": "tslint-config-prettier-check ./tslint.json",
+    "clear-cache": "rm -fr ./.cache ./public"
```
[the remark this answers is not in the document]
Comment (aerfio) [reply package.json]: and you could add `clean` to `build` command for our convienience, e.g.:
```
"build": "npm run clean && gatsby build"
```

^ It's just my thought (but still please change `rm -rf` into `gatsby clean`)

**Comment by aerfio** [comment]
image
On a mobile view I think there's a bit too much space above heading **Service Menagement**

**Comment by aerfio** [comment]
It kinda messes up while resizing

image

File: src/custom.d.ts [hunk 6]
```diff
@@ -16,3 +16,5 @@ declare module "*.js" {
 declare module "prism-react-renderer";
 declare module "react-copy-to-clipboard";
 declare module "react-sticky";
+declare module "react-modal-hook";
+declare module "react-modal";
```
Comment (aerfio) [inline src/custom.d.ts]: https://www.npmjs.com/package/@types/react-modal

File: src/components/shared/Modal/index.tsx [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+      typeof window !== `undefined` && window.innerWidth > sizes.phone
+        ? `rgba(0, 0, 0, 0.48)`
+        : `#fff`,
+  };
+
+  const [showModal, hideModal] = useModal(
+    ({ in: open, onExited }: { in: boolean; onExited: boolean }) => (
+      <>
+        <StyledModal
+          style={{
```
Comment (aerfio) [inline src/components/shared/Modal/index.tsx]: inline style?

Comment (magicmatatjahu) [reply src/components/shared/Modal/index.tsx]: nope

File: src/components/layout/Navigation.tsx (hunk 2 above)
Comment (magicmatatjahu) [reply src/components/layout/Navigation.tsx]: This is temporary comment for future, when we enabled roadmap section for all users.

**Pull request merged**